networkit/cpp/generators/StaticDegreeSequenceGenerator.cpp

```diff
@@ -43,22 +42,54 @@ bool StaticDegreeSequenceGenerator::isRealizable() {
         return false;
     }
 
-    /* Second inequality. */
-    // this inequality needs a sorted sequence
-    std::vector<count> sortedSeq = seq;
-    Aux::Parallel::sort(sortedSeq.begin(), sortedSeq.end(), std::greater<count>());
+    /**
+     * Second inequality
+     * We now check that for all 0 <= j < n the following inequality holds
+     *   sum(d[i] for 0 <= i <= j) !<= sum( min(j+1, d[i]) for j < i < n),
```

Comment:
If understood correctly, the inequality should be `<=` not `!<=`, right?

Reply:
Yes, you are right. The exclamation mark was meant in a \stackrel{!}{\le} sense, i.e. "make sure that this holds". But I see how this is confusion in a C++ context and thus changed it.